### PR TITLE
CORE-1392 Add legacy DE card to Dashboard

### PR DESCRIPTION
**Pull request by ash3rz**
I threw in the `NEW` icon, but I'm open to other ideas.  Also, I tested the dismiss button using the `ipctest` account, so the prompt doesn't currently show there anymore.

Here's what users will see when logged out:
image

And logged in:
image

### CODE REVIEW

**Comment by tinalcv** [comment]
@sri2k1us  @ash3rz 
Thank you, now that I see how things would actually look, I think we can definitely simplify/streamline.  See what you think of the below and maybe we can try to see what it will look like and bring in MMS/others to weigh in?  
Maybe:

= = =
**Logged out screen:**
[some icon/color on the card to draw attn]

The Discovery Environment has a new interface that delivers even more features and functionality to make doing your collaborative computational research in CyVerse a breeze!  Explore how it's now easier and faster to find, launch, and share your saved data, analyses, apps, and contacts.  

You can still use [the old interface][link] until July 5, when we will sunset it. All saved work is reflected in both interfaces, so explore the improved DE with confidence. 
= = =
**Logged in screen:**
[some icon/color on the card to draw attn]

Welcome to the Discovery Environment! We've redesigned the interface so it's even easier and faster to find, launch, and share your saved data, analyses, apps, and contacts.  

You can still use [the old interface][link] until July 5, when we will sunset it. All saved work is reflected in both interfaces, so explore the DE's new features and functionality with confidence. 

New to CyVerse?  Take a Tour by clicking the card below.


 [Question to team: on the Take a Tour card, can we make the "Take a Tour" heading a live link to the tour, delete the "Start Tour" at the bottom but leave the "Dismiss"?]

**Comment by psarando** [comment]
> [Question to team: on the Take a Tour card, can we make the "Take a Tour" heading a live link to the tour, delete the "Start Tour" at the bottom but leave the "Dismiss"?]

I'm sure we can. Should we include both links? Either way would look fine to me for this card.

Also, maybe we should consider making other dashboard card titles clickable as well, with some default action for that card type, such as opening details for app cards (but still keeping the `i` icon at the bottom which also opens the details).

**Comment by sri2k1us** [comment]
> > [Question to team: on the Take a Tour card, can we make the "Take a Tour" heading a live link to the tour, delete the "Start Tour" at the bottom but leave the "Dismiss"?]
> 
> I'm sure we can. Should we include both links? Either way would look fine to me for this card.
> 
> Also, maybe we should consider making other dashboard card titles clickable as well, with some default action for that card type, such as opening details for app cards (but still keeping the `i` icon at the bottom which also opens the details).

While we can make the heading a link, I don't personally like that from the usability standpoint. I like the way it is right now. Agreed can we can make the card itself clickable with default actions associated with it. But for a later PR 😅

**Comment by sri2k1us** [comment]
How about we replace the `New` Icon with `import NewReleasesIcon from '@material-ui/icons/NewReleases';` in gold color ?

**Comment by tinalcv** [comment]
I get it about the usability, so an alternative suggestion in light of @sri2k1us comment is to rename the Tour card title to "Tour the Discovery Environment" and rename "Start Tour" to "Take a Tour".  This will make the wording in the Logged In DE card consistent with the Tour card.  Holler if this is confusing.

**Comment by sri2k1us** [comment]
> I get it about the usability, so an alternative suggestion in light of @sri2k1us comment is to rename the Tour card title to "Tour the Discovery Environment" and rename "Start Tour" to "Take a Tour". This will make the wording in the Logged In DE card consistent with the Tour card. Holler if this is confusing.

Sounds good to me 👍🏽

**Comment by ash3rz** [comment]
Thanks for the feedback!  Screenshots have been updated

**Comment by tinalcv** [comment]
Thanks @ash3rz, so speedy!   I like the new gold icon too, good size and no words/letters. Bravo!

**Review by sri2k1us — APPROVED** [review]
:shipit:

File: src/pages/_app.js [hunk 1]
```diff
@@ -211,6 +214,7 @@ function MyApp({ Component, pageProps }) {
                 fileIdentifiers,
                 vice,
                 grouper,
+                legacyDeUrl: legacyDeUrl,
```
Comment (psarando) [inline src/pages/_app.js:217]:
```suggestion
                legacyDeUrl,
```

**Review by psarando — APPROVED** [review]
LGTM 👍

**Comment by ash3rz** [comment]
Thanks for the reviews!

**Pull request merged**